### PR TITLE
Update spanish_sp.txt

### PR DESCRIPTION
**Pull request by Wirlie**

### CODE REVIEW

**Comment by Wirlie** [comment]
@gdabi you can take a look for these commits?

**Comment by Wirlie** [comment]
@Gymnasiast please merge it. I translated these strings using original translations of RCT2-Spanish. I tested these translations in-game.

But.... @gdabi has not answered my emails from 5 days ago :/

I will contact gdabi when he come back.

File: data/language/spanish_sp.txt [hunk 1]
```diff
@@ -4047,114 +4050,114 @@ STR_5736    :Jugador
 
 ## RCT Original
 <Forest Frontiers>
-STR_SCNR    :Forest Frontiers
-STR_PARK    :Forest Frontiers
-STR_DTLS    :Deep in the forest, build a thriving theme park in a large cleared area
+STR_SCNR    :Fronteras Forestales
```
Comment (Gymnasiast) [inline data/language/spanish_sp.txt]: Were the scenario names translated in the original Spanish RCT1?

File: data/language/spanish_sp.txt [hunk 2]
```diff
@@ -4483,11 +4486,9 @@ STR_PARK    :Fort Anachronism
 STR_DTLS    :
 
 #########
-# Rides #
+# STALL #
 #########
 
-#WW
```
Comment (Gymnasiast) [inline data/language/spanish_sp.txt]: Why did you take this entry out? Please add it back in.

File: data/language/spanish_sp.txt [hunk 3]
```diff
@@ -4483,11 +4486,9 @@ STR_PARK    :Fort Anachronism
...
-[CONDORRD]
-STR_NAME    :Condor Ride
-STR_DESC    :Riding in special harnesses below the track, riders experience the feeling of flight as they swoop through the air in Condor-shaped trains
-STR_CPTY    :4 passengers per car
+[INFOK]
```
Comment (Gymnasiast) [inline data/language/spanish_sp.txt]: Is the original RCT2 description of the Information kiosk incorrect in Spanish? You only need to specify an override if the original is badly translated or plainly wrong.

**Comment by gdabi** [comment]
Sorry guys.

It's OK to use the original translations, but I don't have the original RCT2, so feel free to override my translated strings.

**Comment by Gymnasiast** [comment]
Original strings aren't necessarily better. Since I know hardly anything about Spanish, I assume that you (Wirlie and gdabi) make that judgement, rather than blindly replacing everything with the original translations.

Anyway, I left some remarks on the PR for Wirlie to look at.

**Comment by Gymnasiast** [comment]
Original strings aren't necessarily better. Since I know hardly anything about Spanish, I assume that you (Wirlie and gdabi) make that judgement, rather than blindly replacing everything with the original translations.

Anyway, I left some remarks on the PR for Wirlie to look at.

**Comment by gdabi** [comment]
@Gymnasiast that's true. I remember when I was young and was playing RCT2 thinking that some strings doesn't fit exactly what they want to say, and there is another problem too, that there is some "more technical" strings (like Block brakes) which cannot be translated literally.

Anyway, the original translation is very good (for the spanish case), so we can use it as a start point. Later, when a more flexible toolkit is implemented, will make us possible to generate better translations.

**Comment by Gymnasiast** [comment]
@gdabi : I vastly improved the Dutch translation in this 'toolkit', so it seems flexible enough to me for this purpose. (Although I admit there are better solutions.) The original translations for Dutch, German and some (most?) CJK translations were very bad. Good to see that is not the case for Spanish.
Anyway, I trust you two to know what's best for the Spanish translation.

File: data/language/spanish_sp.txt (hunk 3 above)
Comment (Wirlie) [inline data/language/spanish_sp.txt]: original translation show "Inf**r**omacion" not "Info**r**mación"

File: data/language/spanish_sp.txt (hunk 1 above)
Comment (Wirlie) [inline data/language/spanish_sp.txt]: Yup!

File: data/language/spanish_sp.txt (hunk 2 above)
Comment (Wirlie) [inline data/language/spanish_sp.txt]: Reverted in ba406c4, thanks!

**Comment by Wirlie** [comment]
@Gymnasiast Thanks! I have made the necessary changes. Please take a look.

File: data/language/spanish_sp.txt [hunk 4]
```diff
@@ -4482,12 +4487,11 @@ STR_SCNR    :Fort Anachronism
...
 
-#########
-# Rides #
-#########
+#####################
+# Rides/attractions #
+#####################
 
 #WW
-[CONDORRD]
```
Comment (Gymnasiast) [inline data/language/spanish_sp.txt]: This block is still missing. Please re-add it. (The original English text was wrong, as it described the wrong type of track, therefore the Spanish translation is wrong too.)

Comment (Wirlie) [inline data/language/spanish_sp.txt]: Oh! I understand, thanks! I added again this section.

**Comment by Wirlie** [comment]
Okay, thanks very much for your patience. Now I know how to work with the next updates :)

**Pull request merged**